docs_src/examples/Ch-ExamplesUsbDeviceService.md

```diff
@@ -0,0 +1,367 @@
+# USB Camera Device Service RTSP Streaming Guide
+
+## Contents
+
+[Overview](#overview)  
+[System Requirements](#system-requirements)  
+[How It Works](#how-it-works)  
+[Tested Devices](#tested-devices)  
+[Dependencies](#dependencies)  
+[Get the Source Code](#get-the-source-code)  
+[Deploy the Service](#deploy-edgex-and-usb-device-camera-microservice)  
+[Verify the Service](#verify-service-and-device-profiles)   
+[Adding Devices using REST API](#adding-devices-using-rest-api)  
+[Start Video Streaming](#start-video-streaming)  
+[Shutting Down](#shutting-down)  
+[Optional](#optional)  
+[Troubleshooting](#troubleshooting)  
+[License](#license)
+
+
+## Overview
+The EdgeX usb device service is designed for communicating with USB cameras attached to Linux OS platforms. This guide will help configure and build the usb device service and start streaming video from the USB camera.
+
+This service provides the following capabilities:
+- V4L2 API to get camera metadata
+- Camera status
+- Video stream reference
+- FFmpeg framework to capture video frames and stream them to an RTSP server
+- An embedded [RTSP server](https://github.com/aler9/rtsp-simple-server) server
+## System Requirements
+
+- Intel&#8482; Core&#174; processor
+- Ubuntu 20.04.4 LTS
+- USB-compliant Camera
+
+**Time to Complete**
+
+10-20 minutes
+
+**Other Requirements**
+
+You must have administrator (sudo) privileges to execute the user guide commands.
+
+## Tested Devices
+The following devices have been tested with EdgeX USB Camera Device Service:  
+Note: Results may vary based on camera hardware/firmware version and operating system support.
+<!-- sorted alphabetically -->
+- AUKEY PC-LM1E Webcam
+- HP w200 Webcam
+- Jinpei JW-01B USB FHD Web Computer Camera
+- Logitech Brio 4K
+- Logitech C270 HD Webcam
+- Logitech StreamCam
+
+## Dependencies
+The software has dependencies, including Git, Docker, Docker Compose, and assorted command line tools. Follow the instructions linked here to install any dependency that is not already installed [see here](https://github.com/EdgeX-Camera-Management/device-usb-camera/blob/rtsp_doc/docs/setup.md).
```

Comment:
This is a fork of device-usb-camera. Will this material be contributed to the edgexfoundry repo?

Reply:
yes, the other material will be contributed as well.

Reply:
yes, please change link to official repo.